### DISCUSSION TITLE
I want to add dark mode to my website

### DISCUSSION THREAD

**Discussion by vaniwiki** [combostrap/combo#98] [Ideas]
Like this: https://darkmodejs.learn.uno

But I don't know where to put the script code. ComboStrap isn't as straight forward as it used to be, main.php... But with a hint, maybe I'm able to put it in there. Ideally, I would like more skins/themes, like Dark/Sepia/Blue/Grey modes.

On Mac and iPhone I am using the Noir extension, and that works flawlessly. I wish I could just downright steal its functionality.

**Comment by gerardnico** [comment]
Via a combo template.
You can add php, JavaScript and html.

If it's only css, the dokuwiki user css is also included.

**Reply by vaniwiki** [reply]
Excuse me for asking, but what exactly is a combo template, and how to include this?

**Reply by gerardnico** [reply]
This is it ;)
https://combostrap.com/theme/what-is-a-template-z9tl87hg

**Reply by gerardnico** [reply]
I already send you this link on theme no?

It's in the previous page

https://combostrap.com/howto/how-to-create-your-own-theme-m092ki81

This is where I give details on how to customise or create a template

**Comment by gerardnico** [comment]
Note that with the webcode component you can also add it in a slot that is present every where such as a page header.

**Comment by gerardnico** [comment]
Note also that since bootstrap 5.3, there is a dark mode.


https://getbootstrap.com/docs/5.3/customize/color-modes/

I have NOT worked on it but pr are welcomed for a bootstrap dark mode component ;)

**Reply by vaniwiki** [reply]
That would be great, even one's own skins they mention there... Anyhow, only on the wishlist for me. I struggled to even learn wiki syntax, so I will not venture into trying to make a dark mode component. Hopefully someone comes along! 👍🏼

**Reply by vaniwiki** [reply]
Does this mean there is an easy way to enable Dark Mode for the full website, without a component, and just go all dark...?

**Reply by vaniwiki** [reply]
Of course not, no matter how much I tinker with the code... I realised that 5.3 is a new release, and ComboStrap uses 5.01.

Anyways, we just put our wishes here, who knows, one day it may appear... :)

**Reply by gerardnico** [reply]
Sure but you can add it yourself;)

https://combostrap.com/howto/combostrap-styling-how-to-create-and-add-your-own-bootstrap-stylesheet-50zxd4ai
